package.json

```diff
@@ -8,7 +8,10 @@
     "ace3"
   ],
   "activationCommands": {
-    "atom-workspace": "language-arma-atom:toggle"
+    "atom-workspace": [
+      "language-arma-atom:toggle",
+      "language-arma-atom:open-rpt-file"
```

Comment:
Does that get automatically capitalized? Otherwise it would be nice if we could make it `Open RPT File` instead of current `Open Rpt File`.